docs/application-connector/docs/007-architecture-connector-service.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Connector Service
+type: Architecture
+---
+
+## Overview
+
+Connector Service is responsible for generating client certificates used to secure communication between Kyma and external systems.        
+    
+Creating a new client certificate is the first step of configuring external system represented by Remote Environment. Kyma stores root certificate and serves in this case as a Certificate Authority. In order to make validation possible the Connector Service returns complete certificate chain: created client certificate along with the root certificate. 
+
+## API
```

Comment:
details about API are not part of the component architecture, API should be described as a separate doc type 
https://github.com/kyma-project/community/blob/master/guidelines/content-guidelines/content-strategy.md#documentation-types